### PR TITLE
FW-6123 add site menu with app on "has_app" feature

### PR DESCRIPTION
**Pull request by bronson-c**
### Description of Changes
- Adds a new fixture with a site menu including the "app" section
- Loads that fixture as the default menu if a site has the `has_app` feature
- Tests new functionality

### Checklist
- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable
- [x] ~~Admin Panel has been updated if models have been added or modified~~
- [x] Migrations have been updated and committed if applicable
- [x] ~~Insomnia workspace has been updated if applicable~~

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

File: firstvoices/backend/fixtures/appjson-has_app.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+					"href": "/alphabet",
+					"title": "Alphabet",
+					"transKey": "navigation-alphabet"
+				}, {
+					"href": "/categories",
+					"title": "Categories",
+					"transKey": "navigation-categories"
+				}]
+			},
+      "app": {
```
Comment (bronson-c) [inline firstvoices/backend/fixtures/appjson-has_app.json:77]: Tagging @gmcauliffe to check this new "app" section in the default menu to ensure it's correctly matched with the FE. Thanks!

Comment (gmcauliffe) [reply firstvoices/backend/fixtures/appjson-has_app.json:77]: Thanks for checking 👍 
To match the FE, it should appear as an additional item in the `itemsData` array of `resources` (line 49 along with `keyboards`). The values are as below:
{
		"href": "/apps",
		"title": "Mobile App",
		"transKey": "navigation-apps"
	}

**Review by gmcauliffe — CHANGES_REQUESTED** [review]

File: firstvoices/backend/fixtures/appjson-has_app.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+			},
+			"resources": {
+				"id": "resources",
+				"title": "Resources",
+				"transKey": "navigation-resources",
+				"itemsData": [{
+					"href": "/keyboards",
+					"title": "Keyboards",
+					"transKey": "navigation-keyboards"
+				}, {
```
Comment (bronson-c) [inline firstvoices/backend/fixtures/appjson-has_app.json:53]: new app section begins here

**Review by gmcauliffe — COMMENTED** [review]
All good for the front-end. Cheers 👍

**Review by sahil97 — APPROVED** [review]

**Pull request merged**